schema/playground/launcher.schema

```diff
@@ -20,19 +21,28 @@ type LaunchMeCommandRequest {
     improbable.Vector3f impact_point = 1;
     improbable.Vector3f launch_direction = 2;
     float launch_energy = 3;
+    EntityId player = 4;
```

Comment:
What is this?

Reply:
A component should tell you more or less everything you need to know about some game system.

Reply:
This was already discussed (with Jessica). I like `player` or `originating_player` (with a preference for the latter because it's shorter). I don't think it's too important.

Reply:
But I have no idea what it is by looking at it without reading through a load of systems. Which means it's not a good name in this context.

Reply:
The alternative is that you add comments for what the component is for.